src/core/Object3D.js

```diff
@@ -662,6 +668,20 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 		}
 
+		// SkinnedMesh specific
+
+		if ( this.skeleton !== undefined ) {
```

Comment:
I tried to do this once - it's trickier than it seems, because of how ```toJSON``` is written, it makes it hard to do this without copy-pasting the whole ```toJSON``` function.

Ideally, since ```SkinnedMesh``` extends ```Object3D```, you should just be able to call ```Object3D.prototype.toJSON.call(this, meta)``` - but in practice it's trickier, because ```Object3D.toJSON``` defines a bunch of internal functions - not sure if we need those internal functions for the skeleton, but I have run into cases in the past where this made it difficult without duplicating a lot of code, or restructuring.